client/src/components/ReadingList/ReadingList.scss

```diff
@@ -16,6 +30,30 @@
   }
 }
 
+.reading-list-adder {
+  &__startButton {
+    float: right;
+  }
+  &__form {
+    display: flex;
+    justify-content: right;
+
+    .ant-btn {
+      margin-left: 5px;
+      &[disabled] {
+        &:hover {
```

Comment:
nit: I'd add a comment here explaining what this override is doing. Or you can make it a descriptive mixin, like `@disable-antd-hover` or something